src/main/java/emissary/server/EmissaryServer.java

```diff
@@ -366,6 +366,14 @@ public static void stopServer(final String name, final boolean force, final bool
         }
         LOG.info("Done stopping all places");
 
+        try {
+            MetricsManager.lookup().shutdown();
+        } catch (Exception ex) {
+            LOG.warn("No metrics manager available");
+        }
+
+        RollManager.shutdown();
```

Comment:
Should the roller go before metrics, or does that cause issues? It shouldn't, right??

Reply:
Yeah, I would say rollers should stop right after the places stop.

Reply:
It doesn't cause issues being moved before metrics shutdown. @jpdahlke I can easily add a message saying "Rollers have been closed" if wanted, but with `RollManager.shutdown()` being called, that puts out a log message as is -> `INFO emissary.roll.RollManager -   - Closing all rollers (1)`. If we want two I can add one in, if the existing one is enough  I'll just move RollManager shutdown to before Metrics

Reply:
Ah, if there already is one then that's sufficient.